CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The return type of `call.get()` is now the API type `GetCallResponse` which encapsulates the previous `CallResponse` under the `call` property. [#335](https://github.com/GetStream/stream-video-swift/pull/335)
 - Remove Nuke dependency from the SwiftUI SDK. [#340](https://github.com/GetStream/stream-video-swift/pull/340)
 
+### ✅ Added
+- The SDK now provides a mechanism to manage CallKit integration. [#334](https://github.com/GetStream/stream-video-swift/pull/334)
```

Comment:
Can we re-word this to be more specific? As in - we provide helpers inside the SDK to help with your CallKit integration, sth around those lines.

Reply:
Maybe we should do better marketing here as we are not helping with the CallKit integration rather we are doing it completely. 

WDYT about "The SDK now provides a CallKit integration if needed."?